### PR TITLE
fix: corners not rounded on memos-editor-wrapper

### LINKED ISSUE #1208: memos-editor-wrapper does not have rounded corners
### Describe the bug

The memos-editor-wrapper does not have rounded corners which results in a visible element behind the editor when the background color is changed via the custom CSS themes.

Image

### Steps to reproduce

1. In the settings menu add these two lines of custom CSS
`.memo-list-container {background-color: #5a7684;}`
`.page-container {background-color: #5a7684;}`

2. Return to the home page and view the Memo

### PR DESCRIPTION
**Pull request by fiumad**
Image

### CODE REVIEW

**Comment by fiumad** [comment]
Not sure why the backend check failed. I only modified the /web/src/pages/Home.tsx file

**Comment by hyoban** [comment]
fix #1208 

https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword

**Comment by fiumad** [comment]
> fix #1208 
> 
> 
> 
> https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword

Thanks, I'm not sure why my PR title didn't trigger the link.

File: web/src/pages/Home.tsx [hunk 1]
```diff
@@ -43,7 +43,7 @@ function Home() {
         <Sidebar />
         <main className="memos-wrapper">
           <MemosHeader />
-          <div className="memos-editor-wrapper">
+          <div className="memos-editor-wrapper rounded-lg">
```
Comment (hyoban) [inline web/src/pages/Home.tsx:46]: move to `src/less/home.less` can be better

**Review by hyoban — APPROVED** [review]

**Pull request merged**